qiskit/aqua/operators/list_ops/tensored_op.py

```diff
@@ -18,10 +18,14 @@
 from functools import reduce, partial
 import numpy as np
 
+from qiskit import QuantumCircuit
 from qiskit.circuit import ParameterExpression
```

Comment:
```suggestion
from qiskit.circuit import QuantumCirucit, ParameterExpression
```

Reply:
I edited it to correct the spelling of QuantumCirucit in the suggested change